addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -95,6 +95,16 @@ export function isNotAllowedContent(node) {
     return isArtificialVoidElement(node) || VOID_ELEMENT_NAMES.includes(node.nodeName);
 }
 
+export function isSelectionInHtmlContent(selection) {
```

Comment:
Not sure about the name as, technically, even areas where html content is not supported are still html content themselves... 😅 But I guess `isSelectionInAreaSupportingHtmlContent` is quite verbose. Maybe `isHtmlContentSupportedAtSelection`? Or simply `isHtmlContentSupported`, and the "at selection" is implied by the fact that the `isAvailable` signature is to take a selection. The latter is my favorite option personally.

Note that, ideally, the link and clipboard plugins which currently defines their own `isHtmlContentSupported` should use this one, in the sense that the selector should not be triplicated. I think they can all use the selection (even the drop technically happen at the position of the selection if I'm not mistaken.)